app/services/finance/ecf/ect/statement_calculator.rb

```diff
@@ -6,18 +6,14 @@ module Finance
   module ECF
     module ECT
       class StatementCalculator < Finance::ECF::StatementCalculator
-        def voided_declarations
```

Comment:
I guess we either remove those or amend the controllers instead to read them from here 🤔